docs/pull_request_template.md

```diff
@@ -0,0 +1,47 @@
+<!--
+Make sure to familiarise yourself with the Runtime release process at https://github.com/polkadot-fellows/runtimes?tab=readme-ov-file#release-process 
+and the Runtime release guidelines at https://github.com/polkadot-fellows/runtimes?tab=readme-ov-file#release-guidelines.
+Use this PR template to streamline the release process, communicate on the changes that you are proposing, and provide a clear course of action for network stakeholders.
+-->
+
+## I'm submitting...
+<!---
+REQUIRED:
+Classify the type of PR your are submitting.
+-->
+  - [ ] a priority PR (i.e fixes and patches)
```

Comment:
I was not involved in the process of coming up with these classification, but I do wonder if they cover all corners? 

The latter two are especially vague to me. 

Can we perhaps name 2 real PRs as example for each category?

Reply:
> Can we perhaps name 2 real PRs as example for each category?

I have added links to 2+ PRs for each category in the [runtime release guidelines (draft)](https://docs.google.com/document/d/1jWLPI9JIsr5DobyXblrL6GCwGPe5N0lSmClj5MwMsgc/edit?usp=sharing) for convenience. These are the best examples that come to my mind, so they still need to be confirmed by other Fellows.

Reply:
Agree, these are way too vague, and the parentheticals make them more vague. A "small configuration" or "simple migration" can be very high priority. Also, most people tend to think that their own PR is priority, it's just human nature. Checking a box doesn't make it priority, you just have to convince reviewers that it is.

Reply:
> Checking a box doesn't make it priority, you just have to convince reviewers that it is.

Noted. I have edited the release guidelines and the PR template.